egs/wsj/s5/utils/combine_ali_dirs.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+# Copyright 2016  Xiaohui Zhang  Apache 2.0.
+
+# This srcipt operates on alignment directories, such as exp/tri4a_ali
+
+# Begin configuration section. 
+cmd=run.pl
+extra_files= 
+num_jobs=4
+# End configuration section.
+echo "$0 $@"  # Print the command line for logging
+
+if [ -f path.sh ]; then . ./path.sh; fi
+. parse_options.sh || exit 1;
+
+if [[ $# -lt 3 ]]; then
+  echo "Usage: $0 [options] <ref-data-dir> <dest-ali-dir> <src-ali-dir1> <src-ali-dir2> ..."
+  echo "e.g.: $0 --num-jobs 32 data/train exp/tri3_ali_combined exp/tri3_ali_1 exp_tri3_ali_2"
+  echo "Options:"
+  echo " --extra-files <file1 file2...>   # specify addtional files in 'src-ali-dir1' to copy"
+  echo " --num-jobs <nj>                  # number of jobs used to split the reference data directory."
+  echo " Note, files that don't appear in the first source dir will not be added even if they appear in later ones."
+  echo " Other than alignments, only files from the first src ali dir are copied."
+  exit 1;
+fi
+
+ref_data_dir=$1 # data directory to be used as reference for rearranging alignments
+shift;
+dest=$1;
+shift;
+first_src=$1;
+
+rm -r $dest 2>/dev/null
```

Comment:
rm -r is dangerous.  Better to just remove the things that would be the most likely to interfere, such as (after making the directory), removing `$dest/ali.*.gz` or something like that.